modules/otel-config-multicluster.adoc

```diff
@@ -6,19 +6,107 @@
 [id="gathering-observability-data-from-different-clusters_{context}"]
 = Gathering the observability data from different clusters with the OpenTelemetry Collector
 
-For a multicluster configuration, you can create one OpenTelemetry
-Collector instance in each one of the remote clusters and forward all the telemetry
-data to one OpenTelemetry Collector instance.
+For a multicluster configuration, you can create one OpenTelemetry Collector instance in each one of the remote clusters and then forward all the telemetry data to one OpenTelemetry Collector instance.
 
 .Prerequisites
 
 * The {OTELOperator} is installed.
 * The {TempoOperator} is installed.
-* A TempoStack is deployed on the cluster.
+* A TempoStack instance is deployed on the cluster.
 
 .Procedure
 
-. Create a service account for the OpenTelemetry Collector.
+. You must have the following certificates mounted as secrets in the OpenTelemetry Collector instance. If you don't have them, create them.
```

Comment:
@iblancasa,  :smiley: what's your rough estimated probability of all of these certificates already mounted in the reader's OTEL instance? If it's more likely that the reader has to mount at least some of these certificates, then my suggestion is to keep it as a step with possible exceptions. Here's a suggestion with revised wording of the step:
```suggestion
. Mount the following certificates in the OpenTelemetry Collector instance, skipping already mounted certificates.
```